print3/wsgi.py.in

```diff
@@ -25,9 +25,9 @@ application = loadapp("config:" + configfile, name=None)
 
 
 if __name__ == '__main__':
-     PROXYWMS_PORT = str(os.environ.get('PROXYWMS_PORT', 9001))
+     WSGI_PORT = str(os.environ.get('WSGI_PORT', 9001))
```

Comment:
I wouldn't put a default value, the best is to ensure that the service does not start if the environment value is not set. It can lead to absolutely undebuggable situations otherwise...

Reply:
OK, for no default.